ingress/single-cluster/ingress-custom-grpc-health-check/README.md

```diff
@@ -64,6 +64,8 @@ Below is an example Deployment specification when using a health check proxy. Yo
         - containerPort: 50051        
 ```
 
+> Please note the deployments here use the health_check proxy and sample gRPC applications hosted on `docker.io/`.  You can build and deploy these images into your own repository as well.
```

Comment:
Do you have the code source for the app and proxy if people want to build from source ?

Reply:
its in the other readme.md for this PR at the bottom so i can copy those over